docs/source/getting_started/installation.rst

```diff
@@ -98,6 +98,33 @@ Usage with JupyterHub
 Read the details on our :ref:`JupyterLab on JupyterHub documentation page <jupyterhub>`.
 
 
+Usage with Jupyverse
+--------------------
+
+`Jupyverse <https://github.com/jupyter-server/jupyverse/>`__ is a next-generation Jupyter server based on
+`FastAPI <https://fastapi.tiangolo.com/>`__. It can be used instead of
+`jupyter-server <https://github.com/jupyter-server/jupyter_server/>`__, the Jupyter server installed by default with JupyterLab.
+
+You can install ``jupyverse`` with ``pip``:
+
+.. code:: bash
+
+    pip install jupyverse[auth, jupyterlab]
+
+or with ``mamba``:
+
+.. code:: bash
+
+    mamba install -c conda-forge jupyverse
+    mamba install -c conda-forge fps-auth
+    mamba install -c conda-forge fps-jupyterlab
```

Comment:
Can this be a single run or to these need to be run one by one?

Reply:
Yes, good point.